pkg/sdn/plugin/eventqueue.go

```diff
@@ -0,0 +1,140 @@
+package plugin
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/client/cache"
+)
+
+func NewEventQueue(keyFunc cache.KeyFunc) *cache.DeltaFIFO {
```

Comment:
It's only router and SDN, so if you're confident in the correctness of your implementation and we have enough time / effort to properly test it, then I'm ok to go ahead there.